### PR TITLE
Task router: small changes

### PR DESCRIPTION
**Pull request by stsewd**
Just a little refactor (both tasks pass the version_pk as the first
parameter), and docstring updated.

Don't think this is related to https://github.com/readthedocs/readthedocs.org/issues/7360... but maybe? I couldn't find anything else related to that problem :/

### CODE REVIEW

File: readthedocs/builds/tasks.py [hunk 1]
```diff
@@ -78,19 +77,12 @@ def route_for_task(self, task, args, kwargs, **__):
...
 
     def _get_version(self, task, args, kwargs):
-        if task == 'readthedocs.projects.tasks.update_docs_task':
-            build_pk = kwargs.get('build_pk')
-            try:
-                build = Build.objects.get(pk=build_pk)
-                version = build.version
-            except Build.DoesNotExist:
-                log.info(
-                    'Build does not exist. Routing task to default queue. build_pk=%s',
```
Comment (ericholscher) [inline readthedocs/builds/tasks.py:88]: Why did you remove this check?

Comment (stsewd) [reply readthedocs/builds/tasks.py:88]: Both tasks pass the `version_pk` as the first parameter, so there no need to get the build.
